snippets/common.yml

```diff
@@ -937,6 +937,21 @@ files:
                   console.log(reader.result);
               }
               reader.readAsText(content);
+    objc:
+      code: |
+            NSError *error;
+            NSString *result = [Textile.instance.files data:hash error:&error];
+    swift:
+      code: |
+            var error: NSError?
+            NSString *result = [Textile.instance.files data:hash error:&error];
+            [Textile.instance.files data:hash completion:^(NSString *result, NSError *error) {
+              if (error) {
+                // Do something with this error
+              } else {
+                  // Success!
+              }
+            }];
```

Comment:
Not really sure what all this is other than it's not Swift.

Reply:
oh yeah, ha, meant to flag that one for you. copy paste from the method below it just to start it. no idea what the right way to use that is in swift

Reply:
Ok here's the obj c:
```
  NSString *hash = @"some file hash";
  NSError *error;
  NSString *base64 = [Textile.instance.files data:hash error:&error];
  if (error) {
    // Do something with this error
  } else {
    // Success!
  }
```

and the Swift:
```
    let hash = "some file hash"
    var error: NSError?
    let base64 = Textile.instance().files.data(hash, error: &error)
    if (error != nil) {
      // Do something with this error
    } else {
      // Success!
    }```

Reply:
related to these docs, just opened https://github.com/textileio/go-textile/issues/784